src/irc/core/irc-servers.c

```diff
@@ -475,6 +478,14 @@ static void sig_server_quit(IRC_SERVER_REC *server, const char *msg)
 	g_free(recoded);
 }
 
+static void cap_maxline(IRC_SERVER_REC *server)
+{
+	int maxline = atoi(g_hash_table_lookup(server->cap_supported, CAP_MAXLINE));
```

Comment:
`atoi` is a rather shitty function especially when given bad inputs. We have `parse_uint` in misc.c which wraps strtoul to have more reasonable error handling, go with that imo?

Reply:
cool